### PR TITLE
fix static check of importing the same package multiple times

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateekpandey14@gmail.com>

#### What type of PR is this?
fix static check of importing the same package multiple times

/kind cleanup

#### What this PR does / why we need it:

Go allows importing the same package multiple times, as long as different import aliases are being used.
However, this is very rarely done on purpose. Usually, it is a sign of code that got refactored, accidentally adding duplicate import statements. It is also a rarely known feature, which may contribute to confusion.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by prateekpandey14** [comment]
cc @msau42

File: pkg/registry/core/service/storage/storage.go [hunk 1]
```diff
@@ -68,7 +66,7 @@ func NewGenericREST(optsGetter generic.RESTOptionsGetter, serviceCIDR net.IPNet,
 	}
 
 	statusStore := *store
-	statusStrategy := service.NewServiceStatusStrategy(strategy)
+	statusStrategy := svcreg.NewServiceStatusStrategy(strategy)
```
Comment (msau42) [inline pkg/registry/core/service/storage/storage.go:69]: how about we remove `svcreg` and just use `service`

**Comment by dims** [comment]
/lgtm
/approve

NOTE: this is good that we caught this, but this is likely to happen again. We have a way to force a specific alias to be used for an import, but that may be a hard thing for folks to adhere to (it's only used in test/ tree, see verify-import-aliases.sh and update-import-aliases.sh)

**Pull request merged**